dojo/settings/settings.dist.py

```diff
@@ -703,6 +672,148 @@ def generate_url(scheme, double_slashes, user, password, host, port, path, param
 
 vars().update(EMAIL_CONFIG)
 
+# ------------------------------------------------------------------------------
+# SAML
+# ------------------------------------------------------------------------------
+# For more configuration and customization options, see djangosaml2 documentation
+# https://djangosaml2.readthedocs.io/contents/setup.html#configuration
+# To override not configurable settings, you can use local_settings.py
+SAML2_ENABLED = env('DD_SAML2_ENABLED')
+SAML2_LOGOUT_URL = env('DD_SAML2_LOGOUT_URL')
+if env('DD_SAML2_ENABLED'):
```

Comment:
Probably best to `SAML2_ENABLED` to accommodate users not using env vars. Also the env var is already stored there